RxWebKit/Sources/Rx+WebKit.swift

```diff
@@ -60,4 +60,23 @@ extension Reactive where Base: WKWebView {
         return self.observeWeakly(Bool.self, "canGoForward")
             .map { $0 ?? false }
     }
+	
+	
+	/// Reactive wrapper for `evaluateJavaScript(_:completionHandler:)` method.
+	///
+	/// - Parameter javaScriptString: The JavaScript string to evaluate.
+	/// - Returns: Observable sequence of result of the script evaluation.
+	public func evaluateJavaScript(_ javaScriptString:String) -> Single<Any?> {
+		weak var webViewRef = self.base
```

Comment:
indentation here looks off... are you using 8-space tabs or something like that?

Reply:
I used 4-space tabs.

Reply:
It definitely looks like 8. can you double-check? Also might be better to just use regular 4 spaces so it looks aligned with the rest of the project.

Reply:
4 spaces on top, compared to yours: 
![image](https://user-images.githubusercontent.com/605076/55228497-4965f200-522b-11e9-9cb6-a14828370d5f.png)

Maybe GitHub represents tabs as 8 spaces 🤷‍♂️

Reply:
ok, i'll do that